bindings_swift/Makefile

```diff
@@ -39,20 +46,17 @@ move_gen:
 	# Keep top-level Generated for push-to-swift step where .swift files are
 	# copied over to another repo (xmtp-rust-swift)
 	find include/Generated -type f -name "*.swift" -delete
+	./make_xmtp_rust_swift_sources.sh
```

Comment:
renamed `move_gen` to `bridge` and added the include/ and Source/ generation here so that it can be included in the Build phase.